### ISSUE TITLE
travis job for unittest fail

### ISSUE THREAD

**Issue by Dinaamagdy** [threefoldtech/0-templates#285]
travis job for unittest  deson't work  with the following error 
https://travis-ci.com/threefoldtech/0-templates/builds/94473069

**Comment by zaibon** [comment]
yes all the test on this repo are quite behind and have not been maintained. We'll need to get back on track at some point.
